docs/guide/admin_panel.md

```diff
@@ -1,5 +1,9 @@
 # Administration management
 
+[TEST LINK](../references.md)
+
+[TEST LINK](../reference.md)
+
```

Comment:
```suggestion
```

Reply:
That's part of the test commit, will be removed anyway before merging.